assets/customElements/index.js

```diff
@@ -0,0 +1,2 @@
+import map from "./dialog_map";
+import map_filter_form from "./dialog_auto_form";
```

Comment:
Pas besoin de nommer les imports

```suggestion
import "./dialog_map";
import "./dialog_auto_form";
```